packages/components/tooltip/README.md

```diff
@@ -180,75 +180,51 @@ export {
-| `modifiers.applyStyle.enabled`                | `bool`                                                                                                                                                                         |          |                |                                                                                                                                                                                                          |
-| `modifiers.applyStyle.fn`                     | `func`                                                                                                                                                                         |          |                |                                                                                                                                                                                                          |
-| `modifiers.applyStyle.onLoad`                 | `func`                                                                                                                                                                         |          |                |                                                                                                                                                                                                          |
+| Props                  | Type                                                                                                                                                                              | Required | Default   | Description                                                                                                                                                                                              |
+| ---------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------: | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `children`             | `ReactReactElement`                                                                                                                                                               |    ✅    |           |                                                                                                                                                                                                          |
```

Comment:
Is `ReactReactElement` a correct type?

Reply:
This is generated, so I assume so?

Reply:
I didn't know that it's generated. I'm asking because I don't know whether there is a typo in the name so that the type is `ReactElement` instead or `ReactReactElement` is considered as a correct type.